torii/platform/vendor/xilinx.py

```diff
@@ -430,6 +453,122 @@ def _part(self) -> str:
 		'''
 	]
 
+	# Yosys NextPNR prjxray templates
+
+	_yosys_nextpnr_part_map = {
+		'xc7a35ticsg324-1L':  'xc7a35tcsg324-1',  # Arty-A7 35t
+		'xc7a100ticsg324-1L': 'xc7a100tcsg324-1', # Arty-A7 100t
+
+	}
+
+	_yosys_nextpnr_device = {
+		'xc7a35ti':  'xc7a35t',
+		'xc7a100ti': 'xc7a100t',
+	}
+
+	_yosys_nextpnr_family = {
+		'xc7a': 'artix7',
+		'xc7z': 'zynq7',
+	}
+
+	_yosys_nextpnr_required_tools = [
+		'yosys',
+		'nextpnr-xilinx',
+		'fasm2frames',
+		'xc7frames2bit'
+	]
```

Comment:
Please add an extra line between the end of this  block and the start of the next